notebooks_tsqr/NightLog.ipynb

```diff
@@ -0,0 +1,271 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.        recs = service_adapter.get_messages(limit=limit,
</pre><p>I think we should change both instances of <code>recs</code> to something more descriptive, and differ the narrative from the exposure one. e.g. <code>exposure_records/messages</code> <code>narrative_records/entries</code>    </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-ts/ts_logging_and_reporting/pull/20/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
Agreed.  Its a even worse than the obvious because exposure has that we might want to show for both "messages" and "exposures".